rviz_default_plugins/CMakeLists.txt

```diff
@@ -429,6 +422,7 @@ if(BUILD_TESTING)
   if(TARGET map_display_test)
     target_include_directories(map_display_test PUBLIC ${TEST_INCLUDE_DIRS})
     target_link_libraries(map_display_test ${TEST_LINK_LIBRARIES})
+    ament_target_dependencies(marker_test ${TEST_TARGET_DEPENDENCIES})
```

Comment:
`marker_test` -> `map_display_test`

Reply:
Good catch, addressed in 0ee8721a7c0ca9baf6f2f654b950b70354b8adf9.